### PR TITLE
Add wasm gateway configuration to template

### PR DESCRIPTION
**Pull request by azgabur**
Referenced here: https://github.com/3scale-qe/3scale-tests/blob/main/doc/GATEWAYS.md#wasm-gateway

### CODE REVIEW

**Review by dhlavac — APPROVED** [review]

File: config/settings.local.yaml.tpl [hunk 1]
```diff
@@ -18,6 +18,17 @@ development:  # default dynaconf env
       password: ${MASTER_PASSWORD}
       token: ${MASTER_ACCESS_TOKEN}
     gateway:
+      default:
```
Comment (mganisin) [inline config/settings.local.yaml.tpl:21]: Why?

**Comment by azgabur** [comment]
Moved from `config/settings.local.yaml.tpl` to `settings.yaml.tpl`

File: config/settings.yaml.tpl [hunk 2]
```diff
@@ -19,9 +19,18 @@ default:
     gateway:  # More info at GATEWAYS.md
       default:
         kind: "SystemApicast"
+        httpbin:
```
Comment (pehala) [inline config/settings.yaml.tpl:22]: This should 100% be under WASMGateway section, it doesn't make sense for SystemApicast and for other gateways

Comment (azgabur) [reply config/settings.yaml.tpl]: Addressed

**Comment by azgabur** [comment]
Removed/Moved the comment from `settings.yaml`

**Review by mganisin — APPROVED** [review]

**Pull request merged**